config/default.schema.json

```diff
@@ -502,6 +502,11 @@
 			"type": "boolean",
 			"default": true,
 			"description": "Enables button to new Nuxt School Admin beta-version page"
+		},
+		"FEATURE_OAUTH_LOGIN_ENABLED": {
+			"type": "boolean",
+			"default": false,
+			"description": "Enables button Oauth Login feature"
```

Comment:
feature flags must also be added in the Kubernetes config. 
I think you can make a branch with same name [here](https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/brb/instance_cfg.yml) but better ask @mamutmk5

Reply:
Already added the Kubernetes Config with Paul Poppe and tested it in the dev cluster

Reply:
fetch the button from the servers schools list, there must **no new feature flag being added !!!**

Reply:
Why? We have discussed this together, and it works perfectly.